syntax = "proto3";

package mathgame.protobuf;

message Command {
    enum Type {
        JOIN_ROOM = 0;
        ANSWER = 1;
    }
    Type type = 1;
    string player_id = 2;

    oneof payload {
        Answer answer = 11;
    }
}

message Answer {
}